### PR TITLE
Allow user_medias to be optional.

### LINKED ISSUE #261: Make user_medias optional/override
##### SUMMARY

Currently with the way user_medias works it overrides anything the end-user may have manually configured, even when user_medias isn't set, its defaulting to an empty array which is removing any configuration the user may have done manually. Per the API documentation the user_medias field is optional.

Don't update the user_medias when not set and/or provide an override setting like passwd has.

##### ISSUE TYPE
- Feature Idea

##### COMPONENT NAME

zabbix_user

##### ADDITIONAL INFORMATION

```yaml

```

### PR DESCRIPTION
**Pull request by elrondvega**
##### SUMMARY

Allows user_medias to be optional when not set.

fixes #261

##### ISSUE TYPE

- Feature Pull Request

##### COMPONENT NAME

zabbix_user

##### ADDITIONAL INFORMATION

Sets the default to None and handles when value is None.

```paste below

```

### CODE REVIEW

**Review by D3DeFi — APPROVED** [review]
Yes, something like this :)

**Comment by sky-joker** [comment]
Thank you for this patch!  
Could you please enter a changelog file and add integration tests to [test_zabbix_user](https://github.com/ansible-collections/community.zabbix/blob/main/tests/integration/targets/test_zabbix_user/tasks/main.yml)?

https://docs.ansible.com/ansible/latest/community/development_process.html#changelogs

File: changelogs/fragments/264-allow-user_medias-to-be-optional.yaml [hunk 1]
```diff
@@ -0,0 +1,2 @@
+minor_changes:
+  - zabbix_user - user_medias now defaults to None and is optional.
```
Comment (sky-joker) [inline changelogs/fragments/264-allow-user_medias-to-be-optional.yaml:2]:
```suggestion
  - zabbix_user - user_medias now defaults to None and is optional (https://github.com/ansible-collections/community.zabbix/pull/264).
```

Please add this PR URL to the end.

**Review by sky-joker — APPROVED** [review]
LGTM

Thank you @elrondvega :)

**Pull request merged**